README.md

```diff
@@ -13,6 +13,21 @@
 
 Google's software suite for combinatorial optimization.
 
+This fork add an improved support for Xpress solver and
+add the interface to the open source [Sirius](https://github.com/rte-france/sirius-solver) solver
+developed by [RTE](https://www.rte-france.com/).
```

Comment:
This fork is used for several open-source projects at RTE, including :
- [Antares_Simulator](https://github.com/AntaresSimulatorTeam/Antares_Simulator)
- [Farao community](https://github.com/farao-community) ?
- Other open-source projects ?

Reply:
I propose to merge this PR and eventually change the readme on another one. There is no strong need, in my opinion, to describe what we use it for. 
I'm surprised that this change appears here since it comes from #16 .